projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/RecommendationsMeta.svelte

```diff
@@ -53,6 +54,10 @@
 	 * Start a new image analysis job.
 	 */
 	async function onStartAnalysis() {
+		const $event_name =
+			$isaSummary.status === ISAStatus.Completed ? 'clicked_restart_isa' : 'clicked_start_isa';
+		recordBoostEvent( $event_name, {} );
+
```

Comment:
I think explicitly adding individual events to the buttons themselves is better. Calling the function doesn't mean that a click event invoked it.

I don't have a strong preference on this. Just an observation.

Reply:
True. I'll fix this.